src-tauri/src/commands/config.rs

```diff
@@ -74,6 +75,25 @@ pub async fn is_avx_requirement_met(
   }
 }
 
+#[cfg(any(target_arch = "aarch64"))]
+#[tauri::command]
+pub async fn is_avx_requirement_met(
+  config: tauri::State<'_, tokio::sync::Mutex<LauncherConfig>>,
+) -> Result<bool, CommandError> {
+  let mut config_lock = config.lock().await;
+  match config_lock.requirements.avx {
+    None => {
+      config_lock.requirements.avx = Some(false);
+      config_lock.save_config().map_err(|err| {
+        log::error!("Unable to persist avx requirement change {}", err);
+        CommandError::Configuration(format!("Unable to persist avx requirement change"))
+      })?;
+      Ok(config_lock.requirements.avx.unwrap_or(false))
+    }
+    Some(val) => Ok(val),
+  }
+}

```

Comment:
could probably simplify this to just always return `false` or `true` if you want to bypass it.  ARM is never going to return true for AVX

Reply:
probably a good idea, i wasn't thinking too much when i did this.